flang/lib/Lower/OpenMP.cpp

```diff
@@ -304,9 +304,3 @@ void Fortran::lower::genOpenMPConstruct(
       },
       ompConstruct.u);
 }
-
-void Fortran::lower::genOpenMPEndLoop(
```

Comment:
Can you remove the declaration from the header file as well?

Reply:
Thanks @schweitzpgi for catching this one 👍